packages/core/admin/server/controllers/admin.js

```diff
@@ -96,11 +98,24 @@ module.exports = {
     );
     const isHostedOnStrapiCloud = env('STRAPI_HOSTING', null) === 'strapi.cloud';
 
+    const numberOfAllContentTypes = _.size(strapi.contentTypes);
+    const numberOfComponents = _.size(strapi.components);
+
+    const getNumberOfDynamicZones = () =>
+      pipe(
+        map('attributes'),
+        flatMap(values),
+        sumBy(propEq('type', 'dynamiczone'))
+      )(strapi.contentTypes);
```

Comment:
It might be a matter of personal preference (feel free to ignore), but I would suggest that for arrow functions of more than one line, consider using `{}` and a return statement to better delimit the boundaries of the function :slightly_smiling_face: 

```suggestion
    const getNumberOfDynamicZones = () => {
      return pipe(
        map('attributes'),
        flatMap(values),
        sumBy(propEq('type', 'dynamiczone'))
      )(strapi.contentTypes)
    };
```